Readme.md

```diff
@@ -220,6 +220,30 @@ The `method` is optional. The default value is `link`, while the possible values
 
 When using the `copy` method, the import will take longer.
 
+### Importing and export single collections
+
+_**note:**_ Since directories might have been created by the docker daemon instead of a user,
+root privileges might be necessary.
+
+#### Exporting
+Liquid has to be deployed in order to export collections. When deployed, run
+```
+./liquid exportcollection <collection_name>
+```
+The exported collection data is stored in `<collections_directory>/exported/<collection_name`
+
+#### Importing
+you need paths to the following data in order to import a collection:
+- database: the folder must contain a subfolder called data, containing the database
+- blobs: the folder contains the blobs
+- index: a `.tgz` archive containing the index
+run the following command:
+```
+./liquid importcollection <collection_name> <database_path> <blobs_path> <index_path> [method]
+```
+You can also specify whether data is moved, copied or linked by specifying the method.
```

Comment:
Is there a good reason to support three import/export methods? If so, we need automated tests for all of them as part of the regular test suite for each pull request, otherwise the code is actually dangerous. What if I want to copy-export a live collection and it ends up down the `move` codepath because of a seemingly innocent change?

Reply:
I've added the three option since there were already part of the fuctions from `importfromdocker`. There's (maybe) no good reason to have move, ever I agree. Honestly, since you will be the one who has to write the tests, it's your call 😁 
I'd go for just copy then?!